frontend-react/src/components/USLink.tsx

```diff
@@ -1,13 +1,15 @@
 import React, { AnchorHTMLAttributes } from "react";
 import { Link, NavLink } from "react-router-dom";
 import classnames from "classnames";
+import { escapeRegExp } from "lodash";
```

Comment:
Ah, `lodash` isn't a direct dependency for our project so we should add either `lodash.escapeRegExp` or`lodash` to our package.json.  I'd op for the former just because I've been burned before with their bundle sizes.  I know that lodash was trying to be more modular, though, but ya know...lingering bad tastes :P

Reply:
+1 to the former as well! Or just manually escape those characters :D